uktrade/data-hub-frontend
# Feature/download button

## Description of change

Adds a download button to the investments collection page that allows the user to download the filtered list of projects. 

Text displays showing "You can now download these ${n} projects" as per the existing service.

When there are more than 5000 items in the list, the download button is not visible and the text says "Filter to fewer than 5000 projects to download".

## Test instructions

Navigate to `/investments/projects` and see the download button - click the link to download the results as a CSV file. Apply filters to the collection and the CSV file should also apply those filters (check that the download link includes the filters in the href).

## Screenshots

![Screenshot from 2020-11-24 15-41-28](https://user-images.githubusercontent.com/1234577/100116517-bf40b780-2e6b-11eb-8f63-3120123306ce.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
